src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Uno.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.UI.Xaml;
+
+namespace Uno.UI.Xaml.Controls;
+
+/// <summary>
+/// Contains additional attached properties that provide Uno-specific
+/// behavior for TextBox control.
+/// </summary>
+public static class TextBox
```

Comment:
This may be annoying IMO and can easily cause ambiguity in user code.

Can't we have the attached DP in the original TextBox non-static class? Or have it as a non-attached DP there?

Reply:
@dr1rrb Is there a specific reason why the attached property for `ScrollViewer` was in `Uno.UI.Xaml.Controls` namespace too? That is the reason I went this route, but indeed it feels like it could have been just a plain DP directly on `TextBox` itself 🤔

Reply:
@dr1rrb I see one advantage of putting the property directly on `TextBox` could be that as it is done now, one can't easily use the property in XAML, as you can't combine it with `not_win` - so in that regard this might be easier in terms of usability

Reply:
Honestly I don't have strong opinion on this, if everybody is against it, I'm fine to put it directly on the `TextBox` instead of having a second class for that. Any opinion @jeromelaban and @ebariche ?